### PR TITLE
feat(managehashes): add links and update description label

### PR DESCRIPTION
**Pull request by wescopeland**
This PR makes some minor updates to the managehashes.php page:

* "Description" has been renamed to "File Name".
* Links to the official forum topic and supported game files have been added.

Screenshot 2023-03-15 at 5 56 15 PM

These changes are motivated by feedback from https://discord.com/channels/310192285306454017/1085433484048343050.

### CODE REVIEW

File: public/managehashes.php [hunk 1]
```diff
@@ -47,7 +48,7 @@ function UpdateHashDetails(user, hash) {
             var date = new Date();
             var dateStr = date.getUTCDate() + ' ' + shortMonths[date.getUTCMonth()] + ' ' + date.getUTCFullYear() + ' ' + date.getUTCHours() + ':' + ('0' + date.getUTCMinutes()).slice(-2);
 
-            $('.comment-textarea').parents('tr').before('<tr class="comment system"><td></td><td class="w-full" colspan="3"><div><span class="smalldate">' + dateStr + '</span></div><div style="word-break: break-word">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</div></td></tr>');
+            $('.comment-textarea').parents('tr').before('<tr class="comment system"><td></td><td class="w-full" colspan="3"><div><span class="smalldate">' + dateStr + '</span></div><div style="word-break: break-word">' + hash + ' updated by ' + user + '. File Name: "' + name + '". Label: "' + labels + '"</div></td></tr>');
```
Comment (Jamiras) [inline public/managehashes.php:51]: This is for the live update. You need to make a similar change in `updateHashDetails` for the value written to the database.

Comment (wescopeland) [reply public/managehashes.php:51]: Resolved in https://github.com/RetroAchievements/RAWeb/pull/1437/commits/df444e1c61a7289ea74e7c81ee4d0f0003d79af0

File: public/managehashes.php [hunk 2]
```diff
@@ -88,14 +89,19 @@ function UnlinkHash(user, gameID, hash, elem) {
         <?php
         echo gameAvatar($gameData, iconSize: 64);
 
+        echo "<div class='mt-2 flex flex-col sm:grid sm:grid-cols-2 gap-2'>";
+        echo "<a class='btn py-2 block' href='/linkedhashes.php?g=$gameID'><span class='icon icon-md ml-1 mr-3'>💾</span>Supported Game Files</a>";
+        echo "<a class='btn py-2 block' href='/viewtopic.php?t=$forumTopicID'><span class='icon icon-md ml-1 mr-3'>💬</span>Official Forum Topic</a>";
+        echo "</div>";
```
[the remark this answers is not in the document]
Comment (wescopeland) [reply public/managehashes.php:95]: I am happy to change these to standard anchor tags with very limited styling; I re-used what was already on the game page simply to be consistent. I have no strong opinion either way given this is more of an admin tool. Let me know what you think is best and I'll go with whatever you prefer.

[the remark this answers is not in the document]
Comment (Jamiras) [reply public/managehashes.php:95]: Simple links would be consistent with other pages.

[the remark this answers is not in the document]
Comment (wescopeland) [reply public/managehashes.php:95]: 👍 Addressed in https://github.com/RetroAchievements/RAWeb/pull/1437/commits/89a7bb596ce8ef0a576c8b3b1365c18e51affb39.

**Review by Jamiras — APPROVED** [review]

File: public/managehashes.php [hunk 3]
```diff
@@ -88,14 +89,19 @@ function UnlinkHash(user, gameID, hash, elem) {
         <?php
         echo gameAvatar($gameData, iconSize: 64);
 
+        echo "<div class='mt-2 flex flex-col gap-1'>";
+        echo " <a href='/linkedhashes.php?g=$gameID'>Supported Game Files</a>";
+        echo " <a href='/viewtopic.php?t=$forumTopicID'>Official Forum Topic</a>";
```
Comment (Jamiras) [inline public/managehashes.php:94]: This link should be hidden if the game doesn't have an official forum topic.
```
select ID from GameData where ForumTopicID is null;
```

Comment (wescopeland) [reply public/managehashes.php]: Good catch, fixed.

**Pull request merged**